package.json

```diff
@@ -35,6 +35,6 @@
     "debug": "~1.0.3",
     "express": "3.x",
     "lodash": "^2.4.1",
-    "swagger-ui": "~2.0.18"
+    "strong-swagger-ui": "^20.0.0"
```

Comment:
20.0.0 is intentional. I have picked such high number to prevent possible clashes between strong-swagger-ui and swagger-ui versions, as that would be confusing; and at the same time to allow us to properly follow semver rules (increase major version when a new feature is added, etc.)